rfdetr/util/coco_to_yolo.py

```diff
@@ -0,0 +1,75 @@
+# ------------------------------------------------------------------------
+# LW-DETR
+# Copyright (c) 2024 Baidu. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 [see LICENSE for details]
+# ------------------------------------------------------------------------
+
+"""Utils to convert a yolo dataset to coco format."""
+import os
+import supervision as sv
+
+YOLO_YAML_FILE = "data.yaml"
+NECESSARY_SPLIT_DIRS = ["train", "valid"]
+NECESSARY_DATA_SUBDIRS = ["images", "labels"]
+OPTIONAL_SPLIT_DIR = "test"
+
+def is_valid_yolo_format(dataset_dir: str) -> bool:
```

Comment:
please rename this function to `is_valid_yolo_dataset` and move it to `rfdetr/datasets/yolo.py`